modules/ROOT/partials/plugin-files/plugin-file-list-emoticons.adoc

```diff
@@ -1,7 +1,6 @@
-.Base js file
+.Requires other js files
 [source, js]
 ----
-./plugins/emoticons/plugin.js
 ./plugins/emoticons/js/emojiimages.js
 ./plugins/emoticons/js/emojis.js
```

Comment:
This was misleading before - the integrator does not require both files, only the file matching their `emoticons_database` setting
https://www.tiny.cloud/docs/tinymce/latest/emoticons/#emoticons_database

Reply:
@TheSpyder Can you confirm if we `remove` or `leave` these? I thought that these files had to be included based on our discussion last week? Maybe I misunderstood 🤣 .

Reply:
Adjust them. Note that one or the other is required, depending on which one is chosen through the `emoticons_database` setting.

Reply:
resolved in last commit to branch.